rust-sdk/examples/assignment_details/main.rs

```diff
@@ -0,0 +1,44 @@
+use eppo::AttributeValue;
+
+pub fn main() -> eppo::Result<()> {
+    // Configure env_logger to see Eppo SDK logs.
+    env_logger::Builder::from_env(env_logger::Env::new().default_filter_or("eppo")).init();
+
+    let api_key =
+        std::env::var("EPPO_API_KEY").expect("EPPO_API_KEY env variable should contain API key");
+    let mut client = eppo::ClientConfig::from_api_key(api_key)
+        .assignment_logger(|event| {
+            println!("Logging assignment event: {:?}", event);
+        })
+        .to_client();
+
+    // Start a poller thread to fetch configuration from the server.
+    let poller = client.start_poller_thread()?;
+
+    // Block waiting for configuration. Until this call returns, the client will return None for all
```

Comment:
minor, but I thinking "blocking wait" is the term we're looking for

Reply:
I think "blocking wait" is a noun? I'm using the verb here to describe what's going on.